R/utils_data.R

```diff
@@ -7,10 +7,60 @@
   x
 }
 
+#' Tools for working with row names
+#'
+#' @param x A data frame.
+#' @param var Name of column to use for rownames. For \code{column_as_rownames()}, this argument can be the variable name or the column number.
+#'
```

Comment:
We try to use markdown syntax as much as possible in roxygen documentation.

So, for example, here, instead of `\code{column_as_rownames()}`, you can use ``column_as_rownames()``.